apps/nextjs-website/src/app/[productSlug]/quick-start/page.tsx

```diff
@@ -22,6 +24,28 @@ export type QuickStartGuidePageProps = {
   readonly steps?: ReadonlyArray<Step>;
 } & LayoutProps;
 
+export async function generateMetadata(
+  { params }: ProductParams,
+  parent: ResolvingMetadata
+): Promise<Metadata> {
+  const { shared } = translations;
+  const previousTitle = (await parent).title || shared.siteTitle;
```

Comment:
I often see this line of code. Could be this a function, taking the `ResolvingMetadata` and the `translations` as input and returning the value of the title?
You could also test this function